static/sass/custom/_search-results.scss

```diff
@@ -89,6 +89,33 @@
       margin-top: -0.5rem;
     }
   }
+
+  .p-contextual-menu {
+    &__toggle {
+      text-transform: capitalize;
```

Comment:
Do you need this? This will effect all contextual-menus across the site?

Reply:
I've moved this to be scoped for just for the series filter (previously it was scoped for the whole filter row).